frontend.java

```diff
@@ -0,0 +1,540 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+
+public class frontend {
+	static int loggedIn = 0;
+	static int type = 0; // if type = 1 its retail, if type=2 its agent
+	static ArrayList<String> transactionFile = new ArrayList<String>();
+	public static void main(String[] args) {
+		login();
+	}
+	public static void login(){
+		//Runs on start up
+		
+				//Prompts User for log in
+				if(loggedIn == 0){
+					//Not logged in
+					while(loggedIn == 0){
+						System.out.print("Please log in: Command>");
+						String logInCom = input();
+						//System.out.println(logInCom);
+						/*
+						int total = 0;
+						for (char ch : logInCom.toCharArray()){
+					        total += (int)ch;
+					    }
+						if(total == 537){
+						*/
+						if(logInCom.equals("login")){//this doesnt work. 
```

Comment:
Since this is the login() function, what's the reasoning for prompting to enter the "login" command?